### PR TITLE
Remove Android.mk from internal Gradle task

### PR DESCRIPTION
**Pull request by Sunbreak**
## Summary

`Android.mk`s are no longer used for internal Gradle tasks since [First Round of CMake files for React Android](https://github.com/facebook/react-native/commit/b676ca560d8af686276ac4b88e4feb9c5b7e468d) and [CMake setup for ReactAndroid](https://github.com/facebook/react-native/commit/e3830ddffd9260fe071e0c9f9df40b379d54cf26)

## Changelog

[General] [Fixed] - Remove Android.mk from internal Gradle task

## Test Plan

None

### CODE REVIEW

**Comment by cortinico** [comment]
Thanks for sending this @Sunbreak 
Can you please update those tests that broke with your PR:

```
com.facebook.react.tasks.internal.PrepareBoostTaskTest > prepareBoostTask_copiesMakefile FAILED
    java.lang.AssertionError
        at org.junit.Assert.fail(Assert.java:87)
        at org.junit.Assert.assertTrue(Assert.java:42)
        at org.junit.Assert.assertTrue(Assert.java:53)
        at com.facebook.react.tasks.internal.PrepareBoostTaskTest.prepareBoostTask_copiesMakefile(PrepareBoostTaskTest.kt:42)

com.facebook.react.tasks.internal.PrepareGlogTaskTest > prepareGlogTask_copiesMakefile FAILED
    java.lang.AssertionError
        at org.junit.Assert.fail(Assert.java:87)
        at org.junit.Assert.assertTrue(Assert.java:42)
        at org.junit.Assert.assertTrue(Assert.java:53)
        at com.facebook.react.tasks.internal.PrepareGlogTaskTest.prepareGlogTask_copiesMakefile(PrepareGlogTaskTest.kt:42)

com.facebook.react.tasks.internal.PrepareJSCTaskTest > prepareJSCTask_copiesMakefile FAILED
    java.lang.AssertionError
        at org.junit.Assert.fail(Assert.java:87)
        at org.junit.Assert.assertTrue(Assert.java:42)
        at org.junit.Assert.assertTrue(Assert.java:53)
        at com.facebook.react.tasks.internal.PrepareJSCTaskTest.prepareJSCTask_copiesMakefile(PrepareJSCTaskTest.kt:113)

com.facebook.react.tasks.internal.PrepareLibeventTaskTest > prepareBoostTask_copiesMakefile FAILED
    java.lang.AssertionError
        at org.junit.Assert.fail(Assert.java:87)
        at org.junit.Assert.assertTrue(Assert.java:42)
        at org.junit.Assert.assertTrue(Assert.java:53)
        at com.facebook.react.tasks.internal.PrepareLibeventTaskTest.prepareBoostTask_copiesMakefile(PrepareLibeventTaskTest.kt:42)

95 tests completed, 4 failed
```

**Comment by Sunbreak** [comment]
> Thanks for sending this @Sunbreak Can you please update those tests that broke with your PR:

Done. Could you take another look?

File: packages/react-native-gradle-plugin/src/test/kotlin/com/facebook/react/tasks/internal/PrepareBoostTaskTest.kt [hunk 1]
```diff
@@ -37,13 +37,13 @@ class PrepareBoostTaskTest {
           it.boostVersion.set("1.0.0")
           it.outputDir.set(output)
         }
-    File(project.projectDir, "src/main/jni/third-party/boost/Android.mk").apply {
+    File(project.projectDir, "src/main/jni/third-party/boost/CMakeLists.txt").apply {
```
Comment (cortinico) [inline packages/react-native-gradle-plugin/src/test/kotlin/com/facebook/react/tasks/internal/PrepareBoostTaskTest.kt:40]: Nit: the name of this test is `prepareBoostTask_copiesMakefile`. Can you update it to `prepareBoostTask_copiesCMakefile` (same for all the tests)

**Review by cortinico — APPROVED** [review]

**Comment by cortinico** [comment]
Let's fix the nit and I'll import & merge this

**Comment by Sunbreak** [comment]
> Let's fix the nit and I'll import & merge this

Done

**Pull request closed without merging**